src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -54,7 +58,15 @@ export class BidResult extends React.Component<BidResultProps> {
       )
     } else {
       const bidAgain = SHOW_TIMER_STATUSES.indexOf(this.props.status) > -1
-      const buttonMsg = bidAgain ? `Bid ${this.props.sale_artwork.current_bid.display} or more` : "Continue"
+      let nextBid = this.props.sale_artwork.minimum_next_bid.display
+      if (
+        this.props.status === "RESERVE_NOT_MET" &&
+        this.props.bid.cents > this.props.sale_artwork.minimum_next_bid.cents
```

Comment:
Why is this condition necessary?

Reply:
You mean the entire block? I thought `minimum_next_bid` is independent from `reserve` so there are cases where `minimum_next_bid < current user bid < reserve` so in that case we want to suggest the first increment that is higher than user's current bid (which may or may not be higher than `reserve`)

Reply:
I think what we show on the bid button shouldn't depend on the reserve (i.e. it should always be the `minimum_next_bid_cents` or an increment above your last bidder position). I'm looking now for the logic in force that determines this...

Reply:
I guess I need to copy [this logic](https://github.com/artsy/force/blob/master/src/desktop/collections/bidder_positions.coffee#L42) from force.

Reply:
I would use this [bid button[(https://github.com/artsy/force/blob/master/src/desktop/apps/artwork/components/auction/templates/bid.jade#L5) as a reference.

Looks like it's either grabbing your current max bid (which we get from your lot standing-- or in this case I guess the bidder position you just made? the logic is [here](https://github.com/artsy/metaphysics/blob/master/src/schema/me/lot_standing.js#L43) for what that is) or the first increment, which is based off of the `minimum_next_bid_cents` of the [sale artwork](https://github.com/artsy/metaphysics/blob/master/src/schema/me/lot_standing.js#L43). I think on this view you're not already fetching the bid increments, so may have to do some extra math here.

Reply:
hmm, I was implementing a way to fetch `lot_standings` and looking at the [bid button](https://github.com/artsy/force/blob/master/src/desktop/apps/artwork/components/auction/templates/bid.jade#L5) it seems `sale_artwork.bid_increments.filter(function(i) { return i > (myLastMaxBid || 0) })` and `nextBid = this.props.sale_artwork.increments.filter(d => d.cents > this.props.bid.cents)[0].display` are the same where `this.props.bid.cents` is the bid user just placed in the previous screens when 'reserve is not met'. right?

Reply:
still need to use `nextBid = this.props.sale_artwork.increments.filter(d => d.cents > this.props.bid.cents)[0].display` when **OUTBID** and *bidder_position* is not created because `suggested_next_bid` (which we agreed to use) is only available when *bidder_position* is created

Reply:
🤔 If the bidder position is _not_ created, then I don't think we show a bid amount on the payoff screen at all (and in any case I wouldn't want them to try to bid again for a higher amount if the original amount wasn't taken for a different reason).

Reply:
It happens in the bidder flow when users pick a bid from the select box and it gets outbid by the time user confirms the bid. Then MP immediately returns 'OUTBID' without creating *bidder_position*. right?

Reply:
Changed the logic to use `this.props.sale_artwork.minimum_next_bid` when request receives *OUTBID* from MP without creating the bidder position. In that case relay should refresh the values to reflect current `minimum_next_bid`